lib/version_utils.pm

```diff
@@ -128,7 +128,7 @@ sub is_caasp {
     elsif ($filter eq 'VMX') {
         return get_var('FLAVOR') !~ /DVD/;    # If not DVD it's VMX
     }
-    elsif ($filter =~ /^\d\.\d\+?$/) {
+    elsif ($filter =~ /\d\.\d\+?$/) {
```

Comment:
that was the last thing missing to make the tests work for QAM also. The last verification run is: http://skyrim.qam.suse.de/tests/3314

The changes that I need to make to osd are:

```
QAM-CaaSP-autoyast1 -> hostname=master-api
QAM-CaaSP-autoyast2 -> hostname=master-rm
QAM-CaaSP-autoyast3 -> hostname=mixed
QAM-CaaSP-autoyastmaster -> hostname=master-ay
```

For switching QAM to CaaSP 3.0 I might need to also do:

```
QAM-CaaSP-autoyast4 => hostname=master-add
QAM-CaaSP-autoyast5 => hostname=worker-addrm
create another one => hostname=worker-ay1
````

Thank you very much @kravciak  for your patience working with me on this one. LGTM ;)